src/test/testdata/portfolio_performance__bondora_go_grow.csv

```diff
@@ -1,2 +1,5 @@
-Datum,Wert,Buchungswährung,Typ,Notiz
-2019-01-02,-100,EUR,Einlage,: TransferGoGrow
\ No newline at end of file
+Datum,Wert,Buchungswährung,Typ,Notiz
+2020-01-20,1,EUR,Einlage,: TransferGoGrowMainRepaiment
+2020-01-20,1,EUR,Zinsen,: TransferGoGrowInterestRepaiment
+2020-01-20,-1,EUR,Gebühren,: GoGrowWithdrawalFee
+2020-01-20,-1,EUR,Entnahme,: TransferWithdraw
```

Comment:
Is this actually the correct format for the csv from bondora? This looks more like the output from the script. did I make a mistake during review last year?

@Sh4kE could you weigh in? Is this the CSV format from Bondora Go & Grow?